src/actions/acceptBid.ts

```diff
@@ -18,15 +18,19 @@ export const acceptBid = async (
 
   if (!isVersion2) {
     const zAuction = await getZAuctionV1Contract(signer, zAuctionAddress);
+    const hub = await getZnsHub(config.web3Provider, config.znsHubAddress);
+
+    // For any v1 bid this will always return the default registrar
+    const registrar = await hub.getRegistrarForDomain(bid.tokenId);
 
     const tx = await zAuction.connect(signer).acceptBid(
       bid.signedMessage,
-      bid.bidNonce,
+      bid.bidNonce, // auctionId for legacy bids
```

Comment:
Seems like an unnecessary comment. I don't see any reference to auctionId but here.